src/common/ConfirmationDialogue.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from '@mui/material/Button'
+import Dialog from '@mui/material/Dialog'
+import DialogTitle from '@mui/material/DialogTitle'
+import Box from '@mui/material/DialogContentText'
+import DialogActions from '@mui/material/DialogActions'
+import Typography from '@mui/material/Typography'
+import DialogContent from '@mui/material/DialogContent'
+
+export default function ConfirmationDialogue({
+  showConfirmationDialogue,
+  confirmationModalTitle,
+  confirmationContent,
+  onCancel,
+  onConfirm,
+}: {
+  showConfirmationDialogue: boolean
+  confirmationModalTitle: string
+  confirmationContent: JSX.Element
```

Comment:
As this isn't actually the entire content of the dialog, I'd argue that `errorMessage` or something similar is a clearer name